src/helpers.tsx

```diff
@@ -0,0 +1,50 @@
+import { ErrorType } from './types/ErorTypes';
+import { Todo } from './types/Todo';
+
+export enum FilteredBy {
+  DefaultType = '',
+  All = 'all',
+  Active = 'active',
+  Completed = 'completed',
+}
+
+type FilterType = string;
```

Comment:
You have `FilteredBy` enum, so you don't need this type, as filter shouldn't be a string